app-editors/kakoune/files/kakoune-2021.11.08-gcc12.patch

```diff
@@ -0,0 +1,53 @@
+commit d1ea2ffa600fd2a7b14e415b68ceedba3325c5db
```

Comment:
Please include a link at the top of the patch to the relevant Gentoo bug, any relevant upstream bugs, and the upstream commit.

Reply:
That makes sense, will do!